subworkflows/local/input_check.nf

```diff
@@ -63,13 +63,12 @@ def create_fastq_channel(LinkedHashMap row) {
             if (!file(row.fastq_2).exists()) {
                 exit 1, "ERROR: Please check input samplesheet -> Read 2 FastQ file does not exist!\n${row.fastq_2}"
             }
-            fastq_meta = [ meta, [ file(row.fastq_1), file(row.fastq_2) ] ]
+         fastq_meta = [ meta, [ file(row.fastq_1), file(row.fastq_2) ] ]
         }
+
     }
     return fastq_meta
-}
-
-// Function to get list of [ meta, fasta ]
+}// Function to get list of [ meta, fasta ]
```

Comment:
this looks a bit funky? Is the change corret?

Reply:
I had no intention of changing it, I think this is a typo.